Bottom: 031f53ac74e2f1118ff715b248bae81e563cb768
Top:    a8a85dbd19718bc3f965ef6cbe6083bb84f9733e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-12 01:47:41 +0100

Refresh of hid-common-remove-include-draw

---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index 6bd48c3..0bbd621 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -44,11 +44,9 @@
 #include "action.h"
 #include "crosshair.h"
 #include "data.h"
-#include "draw.h"
 #include "error.h"
 #include "global.h"
 #include "mymem.h"
-#include "draw.h"
 #include "clip.h"
 
 #include "hid.h"
